src/chrome/content/rules/Nugg.ad.xml

```diff
@@ -1,29 +1,99 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://71.nuggad.net/ => https://71.nuggad.net/: (6, 'Could not resolve host: 71.nuggad.net')
-Fetch error: http://mtm.nuggad.net/ => https://mtm.nuggad.net/: Too many redirects while fetching 'https://mtm.nuggad.net/'
+	Connection refused:
+		- buran.nugg.ad
+		- ubergit.office.nugg.ad
+
+	Chain issues:
+		cp.nugg.ad
+		demo.nugg.ad
+		io.nugg.ad
+		mail.nugg.ad
+		office.nugg.ad
+		gate.office.nugg.ad
+		mail.office.nugg.ad
+		svn.office.nugg.ad
+		optout.nugg.ad
+		pulse.nugg.ad
+		svn.nugg.ad
+		test.nugg.ad
+		test2.nugg.ad
+		test3.nugg.ad
+		test4.nugg.ad
+		track.nugg.ad
+		webserver002.nugg.ad
+		wpw.nugg.ad
+
+	Self-signed cert:
+		- foobar.nugg.ad
+
+	Timeout:
+		- io2.nugg.ad
+
+	TLS error:
+		- calendar.nugg.ad
+		- docs.nugg.ad
+		- gmail.nugg.ad
+		- groups.nugg.ad
+		- mod.nugg.ad
+		- s.nugg.ad
+		- sites.nugg.ad
+		- start.nugg.ad
+		- video.nugg.ad
+		
+	Cert mismatch:
+		- analyzer.nugg.ad
+		- blog.nugg.ad
+		- client.nugg.ad
+		- co.nugg.ad
+		- cockpit.nugg.ad
+		- files.nugg.ad
+		- forecast.nugg.ad
+		- ftp.nugg.ad
+		- interact.nugg.ad
+		- mtm.nugg.ad
+		- newsletter.nugg.ad
+		- obs.nugg.ad
+		- old.nugg.ad
+		- piwik.nugg.ad
+		- presse.nugg.ad
+		- pressespiegel.nugg.ad
+		- reporting.nugg.ad
+		- salon.nugg.ad
+		- stats.nugg.ad
+		- survey2.nugg.ad
+		- webserver001.nugg.ad
+		- www2.nugg.ad
+		- www3.nugg.ad
+		- xmas14.nugg.ad
+
+		Redirects to plain:
+			- mtm.nuggad.net
 
 -->
-<ruleset name="nugg.ad" default_off='failed ruleset test'>
+<ruleset name="nugg.ad" >
+
+	<target host="nugg.ad" />
+	<target host="www.nugg.ad" />
+
+	<target host="icinga.nugg.ad" />
 
-	<target host="*.nugg.ad" />
 	<target host="*.nuggad.net" />
 
 	<securecookie host="^\.nuggad\.net$" name="^d$" />
 
 	<rule from="^http:"
 	        to="https:" />
+		<exclusion pattern="^http://mtm\.nuggad\.net/$" />
 
-	<test url="http://www.nugg.ad/" />
-	<test url="http://71.nuggad.net/" />
+	<test url="http://71i.nuggad.net/" />
 	<test url="http://71iv.nuggad.net/" />
 	<test url="http://asv.nuggad.net/" />
 	<test url="http://bei.nuggad.net/" />
 	<test url="http://ci.nuggad.net/" />
 	<test url="http://gwp.nuggad.net/" />
 	<test url="http://ip-s.nuggad.net/" />
 	<test url="http://mtm.nuggad.net/" />
+	<test url="http://mtm.nuggad.net/robots.txt/" />
```

Comment:
Trailing slash should be removed here.
Please have a look at the other issues exposed by Travis.